theme/ckeditor5-table/tableproperties.css

```diff
@@ -16,6 +17,25 @@
 				& .ck-toolbar__items > * {
 					width: 40px;
 				}
+
+				& .ck-button {
```

Comment:
If anything is wrong with the toolbar in RTL, it should be fixed [in the toolbar component (.ck-toolbar_compact)](https://github.com/ckeditor/ckeditor5-theme-lark/blob/i/6107/theme/ckeditor5-ui/components/toolbar/toolbar.css#L62), not here.